docs/conf.py

```diff
@@ -172,16 +171,12 @@ def get_version(*file_paths):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 
-html_theme = 'edx_theme'
+html_theme = 'sphinx_book_theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
```

Comment:
is this line removal necessary to work, if yes then related comment should also be removed.